x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -156,7 +156,7 @@ export interface SystemFilterOptions {
   severity: CaseSeverity[];
   status: CaseStatuses[];
   tags: string[];
-  assignees: Array<string | null> | null;
+  assignees: Array<string | null>;
```

Comment:
I'm assuming that assignees null can be replaced by [] and hoping that if not, a test will fail. If you are aware of any issues with this change, please let me know 🙏